### PR TITLE
LPS-194639 | Master

### PR DESCRIPTION
**Pull request by GergoMathe**
[LPS-194639](https://liferay.atlassian.net/browse/LPS-194639)

### CODE REVIEW

**Comment by GergoMathe** [comment]
ci:test:account:nocompile

**Comment by GergoMathe** [comment]
ci:test:sf

**Comment by GergoMathe** [comment]
ci:test:account:nocompile

File: modules/apps/account/account-test/src/testFunctional/tests/Account.testcase [hunk 1]
```diff
@@ -4050,6 +4050,227 @@ definition {
...
+	@description = "This is a use case for LPS-188650 (TC-2). Headless Admin User can list the ACTIVE Users on a certain API page sorted by given name"
+	@priority = 4
+	test HeadlessAdminCanListActiveUsersWithPagingAndSortingByGivenName {
+		task ("Given: Multiple Users are created") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Users",
+				panel = "Control Panel",
+				portlet = "Users and Organizations");
+
+			for (var letter : list "a,b,c,d,e,f,g,h,i,j,k,l") {
```
Comment (gaborlovasdotcom) [inline modules/apps/account/account-test/src/testFunctional/tests/Account.testcase:4062]: Can you formulate a dedicated macro for this loop?

File: modules/apps/headless/headless-discovery/headless-discovery-test/src/testFunctional/macros/APIExplorer.macro [hunk 2]
```diff
@@ -127,6 +127,98 @@ definition {
...
+					site = "true");
+			}
+
+			Type(
+				locator1 = "OpenAPI#PARAMETER_UNDER_METHOD",
+				method = ${method},
+				parameter = ${parameter},
+				value1 = ${value});
+
+			if (isSet(parameter_two)) {
```
Comment (gaborlovasdotcom) [inline modules/apps/headless/headless-discovery/headless-discovery-test/src/testFunctional/macros/APIExplorer.macro:159]: Can we use better names for these parameters?

Comment (GergoMathe) [reply modules/apps/headless/headless-discovery/headless-discovery-test/src/testFunctional/macros/APIExplorer.macro:159]: Sadly no, as the parameters can differ for every endpoint.

File: modules/apps/account/account-test/src/testFunctional/tests/Account.testcase (hunk 1 above)
Comment (GergoMathe) [reply modules/apps/account/account-test/src/testFunctional/tests/Account.testcase]: Done

File: modules/apps/account/account-test/src/testFunctional/tests/Account.testcase [hunk 3]
```diff
@@ -4050,6 +4050,227 @@ definition {
...
+					userFirstName = ${letter},
+					userLastName = ${letter},
+					userScreenName = ${letter});
+
+				Navigator.gotoBack();
+			}
+		}
+
+		task ("When: Every other one is deactivated") {
+			for (var letter : list "b,d,f,h,j,l") {
```
[the remark this answers is not in the document]
Comment (GergoMathe) [reply modules/apps/account/account-test/src/testFunctional/tests/Account.testcase]: Done.

**Comment by GergoMathe** [comment]
ci:test:sf

**Comment by GergoMathe** [comment]
ci:test:account:nocompile

**Pull request closed without merging**